templates/springboot/http/pom.xml

```diff
@@ -18,9 +18,9 @@
 
   <properties>
     <java.version>11</java.version>
-    <spring-cloud.version>2021.0.5</spring-cloud.version>
-    <spring-native.version>0.12.2</spring-native.version>
-    <compiler-plugin.version>3.10.1</compiler-plugin.version>
+    <spring-cloud.version>2021.0.3</spring-cloud.version>
+    <spring-native.version>0.12.1</spring-native.version>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
```

Comment:
@lkingland the PR overall is great - thanks! Just curious why we are downgrading here.